includes/Entity/class-item.php

```diff
@@ -282,15 +282,29 @@ public function get_author( $property = null ) {
 	 * @return string
 	 */
 	public function get_content() {
-		if ( $this->content ) {
+		$text_len = $this->str_length( $this->content );
+
+		if ( ( 0 === $text_len ) ) {
+			if ( ! empty( $this->summary ) ) {
+				return $this->summary;
+			} elseif ( ! empty( $this->name ) ) {
+				return $this->name;
+			}
+		}
+
+		// If this is set to a comment type or if if the content if less than maximum, always return.
+		if ( 'comment' === $this->response_type || $text_len <= MAX_INLINE_MENTION_LENGTH ) {
 			return $this->content;
-		} elseif ( $this->summary ) {
+		}
+
+		// If there is a summary that isn't empty, return that, if not the name.
+		if ( ! empty( $this->summary ) ) {
 			return $this->summary;
-		} elseif ( $this->name ) {
-			return $this->name;
-		} else {
-			return '';
+		} elseif ( ! empty( $this->name ) ) {
+			return $this - name;
```

Comment:
@dshanske this should be `$this->name`?!!

Reply:
Yes, phpcs...